kratos/utilities/sub_model_parts_list_utility.h

```diff
@@ -148,6 +151,19 @@ class KRATOS_API(KRATOS_CORE) SubModelPartsListUtility
         const std::string& SubModelPartName
         );
 
+    /**
+     * @brief This method returns the id corresponding to the intersection of two combinations of sub_model_parts
+     * @param Key0 the first combination to intersect
+     * @param Key1 the second combination to intersect
+     * @param rColors the map containign the combinations
+     * @return the intersection is a combination
+     */
+    int IntersectKeys(
```

Comment:
I think the keys are std::size_t

Reply:
The colors key is defined as an int (in `IntStringMapType` typedef), I don't know how to show the code lines...
However, I find this class messy, I should review the structure of this utility